src/tests/search.spec.js

```diff
@@ -0,0 +1,77 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import { describe, it } from 'mocha';
+import app from '../index';
+import helpers from '../utils/helper';
+
+chai.use(chaiHttp);
+chai.should();
+describe('Search functionality', () => {
+  it('Should search in database', (done) => {
+    chai.request(app)
+      .get('/api/v1/requests/search?status=pending')
+      .set('Authorization', helpers.createToken(1, 'andelaeagle@gmail.com', true, 'admin'))
+      .set('Content-Type', 'multipart/form-data')
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.a('object');
+        res.body.should.have.property('msg').eql('Search results');
+        done();
+      });
+  });
+  it('Should search as requester', (done) => {
+    chai.request(app)
+      .get('/api/v1/requests/search?status=pending')
+      .set('Authorization', helpers.createToken(1, 'rswaib@gmail.com', true, 'Tadmin'))
+      .set('Content-Type', 'multipart/form-data')
```

Comment:
great job. I'm just wondering why you set the content-type to multipart-formdata, is this content type we'll need to use when accessing this route